packages/ava/src/data/statistics/stdlib/cdf.ts

```diff
@@ -0,0 +1,20 @@
+import { isNil } from 'lodash';
+
+/**
+ * Evaluates the cumulative distribution function (CDF) for a normal distribution at a value x
+ * - Reference to equation 26.2.17 in Abramowitz and Stegun: Handbook of Mathematical Functions
+ * @param mu mean
+ * @param sigma standard deviation
+ * */
+export const cdf = (x: number, mu: number = 0, sigma: number = 1): number => {
+  if (sigma < 0 || [x, mu, sigma].some((value) => isNil(value))) return NaN;
+  // transfer to standard normal distribution
+  const normalX = (x - mu) / sigma;
+  /** probability density function of the standard normal distribution */
+  const Zx = (1 / Math.sqrt(2 * Math.PI)) * Math.exp((-1 * normalX ** 2) / 2);
```

Comment:
Use `**` instead of `Math.pow` because u wrote Python?

Reply:
'Math.pow' is restricted from being used ？

![image](https://user-images.githubusercontent.com/71261480/224596050-0765cd7f-2569-47d0-b0c9-dcf7d46ad229.png)

Reply:
get it, 🤔 a eslint rule